contracts/ArchToken.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+/// @title Archimedes Token
+/// @dev This is the contract for the Archimedes rewards token
+contract ArchToken is ERC20 {
+    
+    constructor() ERC20("Archimedes Token", "ARCH") {}
+
+    /// @notice Mints tokens to a recipient.
+    /// @param to the account to mint tokens to.
+    /// @param amount the amount of tokens to mint.
+    function mint(address to, uint256 amount) external {
+        _mint(to, amount);
```

Comment:
Following up on our discussion (after you submitted the PR request - so it's not pair I get it :-) ) 
We are going to start with not-mintable but burnable 

That probably means that
* The `constructor` should probably get a `Treasury` address, mint 100M tokens and send them to it
* We probably should describe it in the contract comments above
* `mint()` function should explicitly revert with an error message
* `burn()` needs to be implemented (and call the internal `_burn`). Also consider emitting an event when we burn

Reply:
There is not external `mint()` by default - just the internal `_mint`, so as long as we don't add one, it won't be available.

Reply:
Added a test to make sure `mint()` throws an error "not a function"

Reply:
Marking as resolved as we will handle burning in separate PR